.azure/azure-pipelines.release.yml

```diff
@@ -0,0 +1,32 @@
+#
+# Copy release artifacts from the CI pipeline to a GitHub release.
+#
+
+name: 0.$(Date:yyyy).$(Date:MM).$(DayOfMonth).$(Rev:rr).0
+
+trigger: none
+
+resources:
+  pipelines:
+  - pipeline: ci_pipeline
+    source: "CI"
+
+steps:
+  - download: ci_pipeline
+    artifact: artifacts
+    patterns: artifacts/xdp-@(devkit|runtime)-x64/xdp-@(devkit|runtime)-x64*.zip
+  
+  - task: GitHubRelease@1
+    inputs:
+      gitHubConnection: "GitHub - nibanks"
```

Comment:
What's this? And why's my name in it? 😄

Reply:
Our internal XDP repo already had a GitHub account connection using your PAT, so I reused that :)

Reply:
👍🏻